internal/factory/container/sandbox.go

```diff
@@ -0,0 +1,19 @@
+package container
+
+import (
+	types "k8s.io/cri-api/pkg/apis/runtime/v1"
+
+	"github.com/cri-o/cri-o/internal/lib/namespace"
+)
+
+type SandboxIFace interface {
```

Comment:
can you describe what this is an interface for  and the motivation (especially because there are a handful of Sandbox objects littered throughought the project)

Reply:
I have add information please check.

Reply:
I am still missing the motivation. can you describe in the comment why we added this interface instead of directly using Sandbox?